### PR TITLE
Add export wins rejected tab

### PR DESCRIPTION
**Pull request by paulgain**
## Description of change
Adds an export wins rejected tab. A win is rejected after the customer has reviewed the win and doesn't agree with the win contents the lead officer has outlined.

## Screenshots
Screenshot 2024-02-22 at 11 41 54

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'

- [ ] Has the branch been rebased to main?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)

### CODE REVIEW

File: src/client/modules/ExportWins/WinsSentList.jsx [hunk 1]
```diff
@@ -22,23 +27,28 @@ export default () => (
...
                 {
-                  label: 'Export amount',
+                  label: 'Total value: ',
                   value: currencyGBP(item.total_expected_export_value),
                 },
-                { label: 'Date won', value: formatMediumDate(item.date) },
-                { label: 'Date modified', value: '???' },
+                { label: 'Date won: ', value: formatMediumDate(item.date) },
+                { label: 'First sent: ', value: '???' },
+                { label: 'Last sent: ', value: '???' },
```
Comment (paulgain) [inline src/client/modules/ExportWins/WinsSentList.jsx:43]: Once the API work has been completed we can pull these values through.

**Review by elcct — APPROVED** [review]

File: test/a11y/cypress/config/urlTestExclusions.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ export const urlTestExclusions = [
   { url: '/companies/:companyId/exports' },
   { url: '/exportwins/' },
   { url: '/exportwins/sent/' },
+  { url: '/exportwins/rejected/' },
```
Comment (DeanElliott96) [inline test/a11y/cypress/config/urlTestExclusions.js:11]: These both appear here twice, is this correct? The others are on line 146 and 147.

**Review by DeanElliott96 — APPROVED** [review]
Looks good, possible duplicate URL inside the `urlTestExclusions`.

File: test/a11y/cypress/config/urlTestExclusions.js (hunk 2 above)
Comment (paulgain) [reply test/a11y/cypress/config/urlTestExclusions.js]: Good spot, we only need one of each!

**Pull request merged**